src/KeyManager.ts

```diff
@@ -372,6 +374,32 @@ export class KeyManager {
       );
     }
 
+    if (authServerSigningKey) {
```

Comment:
Shouldn't we make this a requirement instead and raise an exception when not provided?

Reply:
Good call.